<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgNewCategory</class>
 <widget class="QDialog" name="dlgNewCategory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="txtCategory">
        <property name="placeholderText">
         <string>Category name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="pteDesc">
        <property name="placeholderText">
         <string>Category description.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
